week-1/explore/chrome.md

```diff
@@ -12,7 +12,12 @@ invoke the function `alert` with an input string of `"Hello world!"`;
 
 What effect does calling the `alert` function have?
 
+// a pop up appears on top of the page that shows the message is given to the alert function
+
 Now try invoking the function `prompt` with a string input of `"What is your name?"` - store the return value of your call to `prompt` in an variable called `myName`.
 
 What effect does calling the `prompt` function have?
+//it opens a dialog box the lets user input a vlaue to get sotred in the variable we already defined , myName .
+
 What is the return value of `prompt`?
+// it reurns undefined , but by chance i gave the variable to console.log(myname) and console returned the variable , myName , value which shows prompt sotred the value to our variable but doesn't return anything by itself .
```

Comment:
I'm not sure what you mean here when you say "value which shows prompt stored the value

> value which shows prompt sotred the value to our variable but doesn't return anything by itself
Could you try clarifying what you meant here - it's good practice to try focusing your language so your technical communication is as precise as possible

Reply:
i was trying to to get dynamic values from users by using prompt , it as it was so complicated so i left it

Reply:
i will check this out